src/fs.ts

```diff
@@ -48,21 +48,17 @@ export async function writeJson(
   data: object,
   cwd = '.',
 ): Promise<void> {
-  try {
-    const text = JSON.stringify(data, null, '  ');
-    return writeText(path, text, cwd);
-  } catch (e) {
-    return;
-  }
+  const text = JSON.stringify(data, null, '  ');
+  return await writeText(path, text, cwd);
 }
 
 type ListOptions = GlobOptions & {
   extensions?: string[];
 };
 
 export async function list(
-  pattern = '**/*',
-  cwd = '.',
+  pattern: string,
+  cwd: string,
   options: ListOptions = {},
 ): Promise<string[]> {
   const { extensions, ...globOptions } = options;
```

Comment:
just calling out that I haven't added any tests around passing any of the `globOptions` here.  There's quite a dew and I'm working on the assumption that the third party library works and doesn't need retesting here.